custom-events/README.md

```diff
@@ -0,0 +1,25 @@
+## Ratatui Custom Events Template
+
+The template will create the following project structure:
```

Comment:
Maybe add some narrative about how the flow of the app works in this example. I.e. the first thing ask / answer is why would I choose this template rather than the other ones?